accepted/0000-timezones.md

```diff
@@ -0,0 +1,103 @@
+
+* **Managed systems timezone:** the timezone of each minion/client. This doesn't have any impact on this RFC, but it's important to mention.
+
+* **Uyuni Server timezone:** the configured timezone in the server where uyuni is deployed 
+
+* **User preferences timezone:** the defined timezone on the user preferences settings. Here we should be able to choose the timezone, date format (yyyyddmm) and the time format (12/24 hours).
+All the shown dates on the UI and operations on them should always assume these user settings. 
+
+* **Client browser timezone:** the browser/pc timezone where the client is accessing from. This timezone shouldn’t affect any displayed date, but the user should be aware of it.
+
+
+# Detailed design
+[design]: #detailed-design
+
+There are 3 topics that we need to have in consideration here:
+
+## Display of dates
+
+The display of dates should always take in consideration the 3 user preferences settings: timezone, date-format and time-format.
+
+**For example:** '2019-09-10 13:00:00 **CEST**'
+
+At the moment only the setting timezone is implemented. The property date-format and time-format could be added on a second iteration, 
+but the code should be prepared to easily support them.
+
+Example:
+
+![timezones](images/timezones2.png)
+
+Ideally, these dates should be formatted server side using the already existing java methods:
+
+```java
+LocalizationService.getInstance().formatCustomDate(date)
+```
+
+For the new stack (spark+react) if you want to convert a JSON to use the correct format, use the following GSON date adapter `UserLocalizationDateAdapter` 
+that will convert your class dates to the user defined format.
+
+The previous gson adapter 'ECMAScriptDateAdapter' that converts dates to UTC should be avoided.
+
+## Manipulation of dates on the browser
+
+Since the browser Date implementation doesn't support timezones, if we ever need to do any date manipulation more than displaying a date
+ we should never use the object Date() directly but the `moment-timezone` library.
+
+This will require a refactor to the component react component `DateTimePicker` to properly manage timezones using the library `moment-timezone`
+instead of manually calculating the diff between the user preference and browser timezone.
+
+## Create awareness to the user regarding the used timezones
+
+The user should always be aware in which timezone his account is configured and from where he is accessing from.
+
+To create context to the user regarding timezones, 2 clocks should be added to the top. One with the user configured timezone and
+the other one with the browser timezone. 
+
+![timezones](images/timezones.png)
+
+
+In cases we are dealing with user operations, for instance, scheduling new actions, more context information should be added.
+
+**Example:** `Applying the highstate has been scheduled at '2019-09-10 13:00:00 CEST', 14 hours from now`
```

Comment:
Where should that be shown? Logs? UI?

Reply:
On the UI. We already show "Applying the highstate has been scheduled". My suggestion would be to add just a bit more info